components/admin_console/schema_admin_settings.jsx

```diff
@@ -350,18 +350,50 @@ export default class SchemaAdminSettings extends React.Component {
 
     buildButtonSetting = (setting) => {
         const handleRequestAction = (success, error) => {
-            setting.action(success, error, this.state['ServiceSettings.SiteURL']);
+            const successCallback = (data) => {
+                const metadata = new Map();
```

Comment:
If you do...
```suggestion
                const metadata = new Map(Object.entries(data));
```
...then you can delete lines 355-357 because [`new Map` takes an optional constructor parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map).